gallery/templates/nav.html

```diff
@@ -26,7 +26,7 @@
                             </ul>
                         </li>
                     {% endif %}
-                    {% if auth_dict['is_eboard'] or auth_dict['is_rtp'] %}
+                    {% if auth_dict['is_eboard'] or auth_dict['is_rtp'] or auth_dict['is_alumni'] %}
```

Comment:
I don't think we should give alumni admin power to enable/disable lockdown.

Reply:
Agreed! LGTM once this is fixed

Reply:
Not sure why I changed that 🤔